### PR TITLE
Tlspe 35 - report of users process should be sorted.

### PR DESCRIPTION
**Pull request by dlhaines**

### CODE REVIEW

File: config/getGrades-TEST-A.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {"Meta":{"Message":"COMPLETED","httpStatus":200},"Result":{"AssignmentInfo":{"AssignmentData":[
 
+{"Score":5013.3,"Published_Score":5013.3,"User_Id":-365167227025976400,"Finished_At":"2017-06-26T12:09:29.107-04:00","Unique_Name":"VillageKitchen"},
```
Comment (zqian) [inline config/getGrades-TEST-A.json:3]: This is identical data line as before in line 5.

File: src/test/java/edu/umich/its/spe/SPESummaryTest.java [hunk 2]
```diff
@@ -34,81 +34,81 @@ public void tearDown() throws Exception {
...
-//	@Test
-//	public void checkToStringFormatLastDateFoundEmpty() {
-//		ss.appendUser("user 1",true).appendUser("user 2",true).appendUser("user 3",true);
-//		String tos = ss.getStoredGradesLastRetrieved().toString();
-//		assertEquals("what?","",tos);
-//	}
-	
+
+	//	@Test
+	//	public void checkToStringWithUsers() {
```
Comment (zqian) [inline src/test/java/edu/umich/its/spe/SPESummaryTest.java:75]: Do we still need those test cases? If not, maybe those could be removed altogether?

File: config/getGrades-TEST-A.json (hunk 1 above)
Comment (dlhaines) [reply config/getGrades-TEST-A.json:3]: The important thing here is the order.

File: src/test/java/edu/umich/its/spe/SPESummaryTest.java (hunk 2 above)
Comment (dlhaines) [reply src/test/java/edu/umich/its/spe/SPESummaryTest.java:75]: I'll review the tests next time I update this test file.

**Pull request merged**